client/post-editor/editor-status-label/index.jsx

```diff
@@ -7,28 +7,21 @@
 import PropTypes from 'prop-types';
 import { localize } from 'i18n-calypso';
 import React from 'react';
+import { connect } from 'react-redux';
 import classNames from 'classnames';
-import Gridicon from 'gridicons';
 
 /**
  * Internal dependencies
  */
 import * as postUtils from 'lib/posts/utils';
 import EditorStatusLabelPlaceholder from './placeholder';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getEditorPostId } from 'state/ui/editor/selectors';
+import { getSitePost } from 'state/posts/selectors';
 
-class StatusLabel extends React.PureComponent {
-	static displayName = 'StatusLabel';
```

Comment:
do we want to keep the `displayName`? after minification, the class name here turns into something like `A`, which makes it appear as `A` in devdocs and in the react dev tools.

Reply:
I don't know: we usually don't add `displayName` to new React components. In development mode, the names are right and in production, the only place where we really need to get the display name right are the devdocs examples.

The [official React best practice](https://github.com/facebook/react/issues/4915#issuecomment-335803765) seems to be that correct display names in devtools in minified production code are not expected to be right: the display names are deemed unnecessary code that doesn't need to be in the production build.